lib/inv_cg_quda.cpp

```diff
@@ -361,7 +348,7 @@ namespace quda {
 
     // compute initial residual
     double r2 = 0.0;
-    if (param.use_init_guess == QUDA_USE_INIT_GUESS_YES) {
+    if (advanced_feature && param.use_init_guess == QUDA_USE_INIT_GUESS_YES) {
```

Comment:
Why does this happen only if we're using an `advanced_feature`?

Reply:
Since I want to make sure the schwarz preconditioner does not use initial guess ...

Reply:
Then shouldn't it be `!advanced_feature`?

Reply:
Normally `advanced_feature` would be `true`, and for schwarz it will be `false`.

Reply:
Understood, thanks